doc/Language/modules.pod6

```diff
@@ -524,6 +524,65 @@ The directories in the C<RAKULIB> path will be searched for modules when
 Raku C<need>s, C<use>s or C<require>s them. Directories that start with a
 dot are ignored and symlinks are followed.
 
+=head1 Testing modules
```

Comment:
Maybe testing distributions, as opposed to modules, would be better here? You're talking about testing modules in distributions, not individual modules.

Reply:
added clarification and 'distribution' to title